lcd_test/helpers.go

```diff
@@ -262,17 +265,11 @@ func defaultGenesis(config *tmcfg.Config, nValidators int, initAddrs []sdk.AccAd
 	var distrData distr.GenesisState
 	cdc.MustUnmarshalJSON(genesisState[distr.ModuleName], &distrData)
 
-	commPoolAmt := sdk.NewInt(10)
+	// TODO: Fix this so that there are 10 tokens in the module account
```

Comment:
Do we want to fix this now or in another PR?

Reply:
Now is good, just needs to get fixed. Didn't have time to dig into that in detail earlier this week.